lib/resources/r4/document_reference.yaml

```diff
@@ -2,13 +2,99 @@
+        info_link: http://hl7.org/fhir/r4/valueset-composition-status.html
+
 - name: type
-  required: 'No'
+  required: 'Yes'
   type: CodeableConcept
-  description: Precise type of clinical document
+  action:
+    - create
+  description: Specifies the particular kind of document referenced.
+  note: The type must include a LOINC or a proprietary coding but not both together. Multiple LOINC codings or a single proprietary coding can be provided.
+        <br/><br/>
+        When providing proprietary code system, it should be of format 'https://fhir.cerner.com/&lt;your EHR source id&gt;/codeSet/&lt;code set&gt;' (where code set is a Millennium code set). Example&#58; 'https://fhir.cerner.com/eb2384f8-839e-4c6e-8b29-18e71db1a0b1/codeSet/72'.
+        <br/><br/>
+        Please note that the <a href="#terminology-bindings">list in the terminology bindings section</a> is not the complete list mapped for our sandbox. Our sandbox has many codes, but we only document a few to provide an example. The list does not represent what would be mapped for one of our client’s domains. This is one of the things we evaluate and implement as needed when we are making apps available at client sites.
+        <br/><br/>
+        Please <a href="https://groups.google.com/d/forum/cerner-fhir-developers">contact us</a> if there is a code you are interested in consuming, and we can evaluate adding it to our sandbox.
+  url: http://hl7.org/fhir/r4/documentreference-definitions.html#DocumentReference.type
+  example: |
+    {
+      "type": {
+        "coding": [
+          {
+            "system": "http://loinc.org",
+            "code": "83805-2",
+            "userSelected": false
+          }
+        ],
+      }
+    }
   binding:
-    description: Precise type of clinical document
+    description: Specifies the particular kind of document referenced.
     terminology:
+    - display: LOINC Document Types
+      system: http://loinc.org
+      info_link: http://hl7.org/fhir/DSTU2/valueset-c80-doc-typecodes.html
+      values:
+      - 18842-5 - Discharge Summary
+      - 11488-4 - Consult Note
+      - 11506-3 - Provider-unspecified Progress Note
+      - 34840-9 - Rheumatology Note
+      - 34839-1 - Rheumatology Consult Note
+      - 68608-9 - Summary Note
+      - 34133-9 - Summary of episode Note
+      - 78280-5 - Surgery Education Note
+      note: Please note that the following is not the complete list of terminology bindings mapped for our sandbox. Our sandbox has many codes, but we only document a few to provide an example. This list also does not represent what would be mapped for one of our client’s domains. This is one of the things we evaluate and implement as needed when we are making apps available at client sites.
+            <br/><br/>
+            Please <a href="https://groups.google.com/d/forum/cerner-fhir-developers">contact us</a> if there is a code you are interested in consuming, and we can evaluate adding it to our sandbox.
```

Comment:
Actually, let's remove this for now.  We'll update it later.

Reply:
You can find the changes here: https://github.com/cerner/fhir.cerner.com/pull/446/commits/1560e66bcece049dcfee1e781105538fe4d7898f